libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -57,11 +65,11 @@
 
     public static final String OBJECT_COUNTRY_CODES = "object-country-codes";
 
-    public static final String CHECKSUM = "checksum";
+    public static final String CHECKSUM = "klv-packet-checksum";
 
     public static final String IMAGE_COORDINATE_SYSTEM = "image-coordinate-system";
 
-    public static final String IMAGE_SOURCE_SENSOR = "image-source-sensor";
+    public static final String IMAGE_SOURCE_SENSOR = Isr.SENSOR_ID;
 
     public static final String TARGET_WIDTH = "target-width";
```

Comment:
Changed variable name and attribute name to TARGET_WIDTH_METERS and target-width-meters.